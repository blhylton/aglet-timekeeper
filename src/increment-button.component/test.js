import React from 'react';
import renderer from 'react-test-renderer';
import IncrementButton from '.';

it('renders correctly', () => {
  const tree = renderer.create(
    <IncrementButton
      decrement={() => {}}
      increment={() => {}}
      initialMs={10000}
      duration={6}
      unit={'seconds'}
      markButtonForDeletion={() => {}}
    />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
