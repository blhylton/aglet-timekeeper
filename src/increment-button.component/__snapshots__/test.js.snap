// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container false false"
>
  <button
    className="button incrementButton"
    onClick={[Function]}
  >
    <div
      className="arrow"
    />
  </button>
  <div
    className="textWrapper"
  >
    <span
      className="duration"
    >
      6
    </span>
    <span
      className="unit"
    >
      s
    </span>
  </div>
  <button
    className="button decrementButton"
    onClick={[Function]}
  >
    <div
      className="arrow"
    />
  </button>
</div>
`;
